### ISSUE TITLE
Prioritize and Send Usability Test Task List

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#1500]
Prioritize the tasks listed in this document:

https://docs.google.com/document/d/1IMcIuQs6yzB36Z1goG35ccSOFR18C_We5MZLlQTwAzw/edit#heading=h.rtvh8tmx1rxz

And share them with the MSU UARC team for review.

**Comment by mbakeryo** [comment]
@jmcglone I made a first pass at prioritizing the tasks and put them at the bottom of our notes: https://docs.google.com/document/d/1IMcIuQs6yzB36Z1goG35ccSOFR18C_We5MZLlQTwAzw/edit#. Some of them aren't so much tasks as outcomes I'd like answers to, so they probably can't be included. My understanding from yesterday's meeting is that the sessions can only manage 4-6 tasks. Anyway, spoiler alert: I think our focus should be on the ebook reader.  Interested in your thoughts.
